cms/djangoapps/contentstore/views/item.py

```diff
@@ -294,6 +294,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
+            is_library_view = view_name == 'container_child_preview'
```

Comment:
How do we know that only libraries will have the view name "container_child_preview"? Won't that be the case for any child of a container view (for instance, components inside of content experiments/split_test_module)?